misc/swagger-ballerina/modules/swagger-to-ballerina-generator/src/main/resources/templates/skeleton/skeleton.mustache

```diff
@@ -0,0 +1,139 @@
+package {{apiPackage}};
+
+import ballerina.net.http;
+@http:configuration {basePath:"{{#each servers}}{{this.url}}{{/each}}"}
+service<http> {{cut info.title " "}} {
+{{#paths}}
+{{#value}}
+{{#get}}
+    @http:resourceConfig {
+        methods:["GET"],
+        path:"{{key}}"
+    }
+    resource {{operationId}} (http:Request req, http:Response resp{{#parameters}}{{>pathParams}}{{/parameters}}) {
```

Comment:
Shall we migrate to the newer resource syntax with Connection and InRequest in the signature unless any special reason?

Reply:
Fixed with 8f4d68b